src/assets/workflows.json

```diff
@@ -29,7 +31,7 @@
                 "options":
                 [
                     {
-                        "text":"Click on the map",
+                        "text":"Select a delineation point on the map.",
```

Comment:
"Select a delineation point on a blue stream line on the map." (this clarifies what a "delineation point" is and is is similar to the current StreamStats text "Use your mouse or finger to click or tap a blue stream cell on the map").